src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -185,20 +209,22 @@ class PaymentMethodList extends Component {
     renderItem({item}) {
         if (item.type === MENU_ITEM) {
             return (
-                <MenuItem
-                    onPress={item.onPress}
-                    title={item.title}
-                    description={item.description}
-                    icon={item.icon}
-                    disabled={item.disabled}
-                    iconFill={item.iconFill}
-                    iconHeight={item.iconSize}
-                    iconWidth={item.iconSize}
-                    badgeText={this.getDefaultBadgeText(item.isDefault)}
-                    wrapperStyle={item.wrapperStyle}
-                    shouldShowSelectedState={this.props.shouldShowSelectedState}
-                    isSelected={this.props.selectedMethodID === item.methodID}
-                />
+                <OfflineWithFeedback onClose={() => this.dismissError(item)} pendingAction={item.pendingAction} errors={item.errors} errorRowStyles={styles.ph6}>
```

Comment:
```suggestion
                <OfflineWithFeedback onClose={() => this.dismissError(item)} pendingAction={item.pendingAction} errors={item.errors} errorRowStyles={[styles.ph6]}>
```

I think this needs to be an array since its defined [here](https://github.com/Expensify/App/blob/2c391e4b96de35f19241fd93252e0fdccb7687bd/src/components/OfflineWithFeedback.js#L47-L48) as an array. I get this error when testing:

![Screen Shot 2022-08-11 at 4 11 23 PM](https://user-images.githubusercontent.com/12268372/184130537-e23e61e0-e7ee-42df-944c-03308d06a15b.png)

Reply:
I left a comment about this, this PR changed the allowed prop types (but I think it should not)

Reply:
I needed to update this to use `StyleUtils.combineStyles` - since we allow errorRowStyles to be either an object or an array, it started throwing an error since we were trying to use ...errorRowStyles when it was just an object